src/main/java/se/techinsight/urlshortener/UrlController.java

```diff
@@ -21,47 +20,47 @@
 @Tag(name = "Url Controller", description = "API for working with URL Controller")
 @RestController
 @Validated
-@RequestMapping({"/api/url"})
+@RequestMapping(value = {"/api/url"})
 public class UrlController {
 
     @Autowired
     private UrlShortenerService service;
```

Comment:
I would prefer constructor injection. If you have a lombok: @RequiredArgsConstructor + private final UrlShortenerService service;

Reply:
ok, will do it